src/Administration/Resources/app/administration/src/module/sw-flow/page/sw-flow-detail/index.js

```diff
@@ -305,25 +305,25 @@ export default {
             return State.commit('swFlowState/setFlow', flow);
         },
 
-        getDetailFlow() {
+        async getDetailFlow() {
             this.isLoading = true;
-            Shopware.State.dispatch('swFlowState/fetchTriggerActions');
 
-            return this.flowRepository
-                .get(this.flowId, Context.api, this.flowCriteria)
-                .then((data) => {
-                    State.commit('swFlowState/setFlow', data);
-                    State.commit('swFlowState/setOriginFlow', cloneDeep(data));
-                    this.getDataForActionDescription();
-                })
-                .catch(() => {
-                    this.createNotificationError({
-                        message: this.$tc('sw-flow.flowNotification.messageError'),
-                    });
-                })
-                .finally(() => {
-                    this.isLoading = false;
+            try {
+                const [, data] = await Promise.all([
+                    Shopware.State.dispatch('swFlowState/fetchTriggerActions'),
+                    this.flowRepository.get(this.flowId, Context.api, this.flowCriteria),
+                ]);
```

Comment:
@ennasus4sun shouldn't this be changed like discussed in [this comment](https://github.com/shopware/shopware/pull/7143/files#r1991783890)? :thinking: